### PR TITLE
Fix some typos in comment

### PR DESCRIPTION
**Pull request by JoeWrightss**
1. "scheduable" to "schedulable".
2. "SynchornizedBeforeSuite" to "SynchronizedBeforeSuite".

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/aws/README.md [hunk 1]
```diff
@@ -142,4 +142,4 @@ If you'd like to scale node groups from 0, an `autoscaling:DescribeLaunchConfigu
...
 - By default, cluster autoscaler will wait 10 minutes between scale down operations, you can adjust this using the `--scale-down-delay-after-add`, `--scale-down-delay-after-delete`, and `--scale-down-delay-after-failure` flag. E.g. `--scale-down-delay-after-add=5m` to decrease the scale down delay to 5 minutes after a node has been added.
-- If you're running multiple ASGs, the `--expander` flag supports three options: `random`, `most-pods` and `least-waste`. `random` will expand a random ASG on scale up. `most-pods` will scale up the ASG that will scheduable the most amount of pods. `least-waste` will expand the ASG that will waste the least amount of CPU/MEM resources. In the event of a tie, cluster autoscaler will fall back to `random`.
+- If you're running multiple ASGs, the `--expander` flag supports three options: `random`, `most-pods` and `least-waste`. `random` will expand a random ASG on scale up. `most-pods` will scale up the ASG that will schedulable the most amount of pods. `least-waste` will expand the ASG that will waste the least amount of CPU/MEM resources. In the event of a tie, cluster autoscaler will fall back to `random`.
```
Comment (bskiba) [inline cluster-autoscaler/cloudprovider/aws/README.md]: > `most-pods` will scale up the ASG that will schedulable the most amount of pods

This sentence doesn' t make sense. Can you fix it while you're at it?

Comment (JoeWrightss) [reply cluster-autoscaler/cloudprovider/aws/README.md]: Thank you for guiding me, @bskiba🐯.

**Comment by bskiba** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**